rs not trimmed properly

In the following command, the rs does not get trimmed properly (assume it's intended properly, I can't figure out how GitHub markdown works):

> ;compile rust
> \`\`\`rs
> fn absoluteValue(num: i64) -> i64 {
>    return if num > 0 {
>       num
>    } else {
>       -num
>    }
> }
> 
> fn main() {
>    println!("{}", abs(-2));
>    println!("{}", abs(2));
> }
> \`\`\`

Leading to the following bizarre error message:
```
error: expected one of `!` or `::`, found `fn`
 --> prog.rs:2:1
  |
1 | rs
  |   - expected one of `!` or `::` here
2 | fn absoluteValue(num: i64) -> i64 {
  | ^^ unexpected token

error: aborting due to previous error
```
Meaning it thinks line 1 is `rs` and line 2 is the function signature.

The correct error message is this:
```
error[E0425]: cannot find function `abs` in this scope
  --> prog.rs:11:19
   |
11 |    println!("{}", abs(-2));
   |                   ^^^ not found in this scope

error[E0425]: cannot find function `abs` in this scope
  --> prog.rs:12:19
   |
12 |    println!("{}", abs(2));
   |                   ^^^ not found in this scope

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0425`.
```
This issue is not present if `rust` is used or if the error made is fixed (renaming `absoluteValue` to `abs`).

--- a/src/commands/utils/CompilationParser.js
+++ b/src/commands/utils/CompilationParser.js
@@ -125,7 +125,7 @@ export default class CompilationParser {
         "mel", "mercury", "mipsasm", "mizar", "mojolicious", "monkey", "moonscript", "n1ql",
         "nginx", "nimrod", "nix", "nsis", "objectivec", "ocaml", "openscad", "oxygene",
         "parser3", "perl", "pf", "php", "pony", "powershell", "processing", "profile",
-        "prolog", "protobuf", "puppet", "purebasic", "python", "py", "q", "qml", "r", "rib",
+        "prolog", "protobuf", "puppet", "purebasic", "python", "py", "q", "qml", "r", "rs", "rib",
         "roboconf", "routeros", "rsl", "ruby", "ruleslanguage", "rust", "scala", "scheme",
         "scilab", "scss", "shell", "smali", "smalltalk", "sml", "sqf", "sql", "stan", "stata",
         "step21", "stylus", "subunit", "swift", "taggerscript", "tap", "tcl", "tex", "thrift",

